.eslintrc.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   "rules": {
     "array-bracket-newline": ["error", { "multiline": true }],
     "arrow-parens": ["error", "as-needed", { "requireForBlockBody": true }],
-    "default-param-last": "off",
+    "default-param-last": "error",
```

Comment:
Hmm for this change, it is not mentioned in the original issue. As such, could you also describe the change in the "Overview of changes" section of the PR description? Similarly, since this is part of airbnb config, we can safely remove this line